tests/end_to_end/test_slack_integration/data/playbooks/generate_events.yaml

```diff
@@ -0,0 +1,44 @@
+- name: Generate events
+  hosts: wazuh-manager
+  vars:
+    alerts_path: /var/ossec/logs/alerts/alerts.json
+    integrations_log: /var/ossec/logs/integrations.log
+  become: true
+  tasks:
+
+    - name: Truncate integrations log
+      shell: echo "" > {{ integrations_log }}
+
+    - name: Attempt a brute force SSH attack
+      expect:
+        command: ssh {{ item }}@localhost
+        timeout: 5
+        responses:
```

Comment:
We must take into consideration clean environments and the fingerprint.

Reply:
Done in 6c4f812